package-lock.json

```diff
@@ -17378,6 +17405,11 @@
                 "yaml-ast-parser-custom-tags": "0.0.43"
             },
             "dependencies": {
+                "jsonc-parser": {
+                    "version": "2.3.1",
```

Comment:
`vscode-jsonrpc` is pinned to the older version...

Reply:
Looks like `yaml-language-server` wants version `2.2.1` but for some reason npm is installing `2.3.1` as a peer dependency. Very strange.